packages/react-reconciler/src/ReactFiberThrow.js

```diff
@@ -174,7 +174,9 @@ function attachPingListener(
       renderExpirationTime,
     );
     if (enableSchedulerTracing) {
-      ping = Schedule_tracing_wrap(ping);
+      if (thenable.__reactDoNotTraceInteractions !== true) {
+        ping = Schedule_tracing_wrap(ping);
```

Comment:
The ping part is a bit weird because it means that means that if we’re only suspended on this promise for the first commit I believe it will decrement and complete the interaction *before* we commit.

I think this only happens if this also happened to cause a “Delayed” suspended state (bad loading) since otherwise we don’t suspend the commit and it’ll continue until it commits.

However fixing this is tricky since we’d have to keep track of this promise and call it once we commit to detach.

Reply:
> The ping part is a bit weird because it means that means that if we’re only suspended on this promise for the first commit I believe it will decrement and complete the interaction before we commit.

Hm... I don't think this is the case, but maybe you can point out what I'm overlooking? We only decrement in `finishPendingInteractions`, which runs after commit.

Reply:
Another way to put it is, if this is ok, then it should be ok to remove this wrapper in all cases.

Because this ping is only relevant when there is an in progress render. If the in progress render keeps its own interactions alive then we never need this.

Reply:
Hm...yeah, maybe wrapping the ping listener is never necessary.